packages/geo-tools/package.json

```diff
@@ -32,6 +32,23 @@
     "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
     "test": "mocha --opts ./mocha.opts \"./lib/test/**/*.test.js\""
   },
+  "eslintConfig": {
```

Comment:
Is there a way to `sort-package-json` to not put the configs above the dep sections (devDeps, peerDeps, and deps) ?  I always dislike having the configs not start after that, it's hard to ever see the dependencies of a particular package. Plus the configs are something that's mostly only important for the devs of the package not the consume.

I could get over it if there isn't a way though :)

Reply:
There is a way to specify the order, but I was worried the order of the fields might also cause an issue with the publish step. Waiting to hear back from rush team.